Module cannot be found error

When this module is used in context of another module (so simply by running `npm install -g <some-package>`), I get the following error:
```Error: Cannot find module '<...>\node_modules\table\dist\index.js'. Please verify that the package.json has a valid "main" entry```

I see that there is a file called index.js in the folder `dist\src`, so maybe some settings for package generation are wrong?
I expect commit d58adb17d4d976b4090c2c629d8b8d35334436ba broke it...

--- a/package.json
+++ b/package.json
@@ -56,10 +56,10 @@
     "ansi"
   ],
   "license": "BSD-3-Clause",
-  "main": "./dist/index.js",
+  "main": "./dist/src/index.js",
   "files": [
-    "dist/**/*.js",
-    "dist/**/*.d.ts"
+    "dist/src/**/*.js",
+    "dist/src/**/*.d.ts"
   ],
   "name": "table",
   "repository": {

